gateway/src/main/scala/filodb/gateway/GatewayServer.scala

```diff
@@ -90,15 +90,15 @@ object GatewayServer extends StrictLogging {
     val sourceConfig = ConfigFactory.parseFile(new java.io.File(userOpts.sourceConfigPath()))
     val numShards = sourceConfig.getInt("num-shards")
 
-    val dataset = settings.datasetFromStream(sourceConfig)
+    val histogramDataset = Dataset("prometheus", Schemas.promHistogram)
```

Comment:
Hardcoding this seems incorrect.

Reply:
Made changes to use config. Please check